src/wrappers.jl

```diff
@@ -33,13 +35,23 @@ CommonRLInterface.actions(w::MyActionWrapper) = [-1, 1]
 actions(w) # will now return [-1, 1]
 ```
 """
-abstract type AbstractWrapper end
+abstract type AbstractWrapper <: AbstractEnv end
 
 """
-Indicate what the wrapped environment is for an AbstractWrapper (see the AbstractWrapper docstring)
+Indicate what the wrapped environment is for an AbstractWrapper (see the AbstractWrapper docstring).
 """
 function wrapped_env end
 
+"""
+Returns the recursively unwrapped environment.
+"""
+function base_env(env::AbstractWrapper)
+    while env isa AbstractWrapper
+        env = wrapped_env(env)
+    end
+    return env
+end
```

Comment:
```suggestion
base_env(env::AbstractWrapper) = base_env(wrapped_env(env))
```
When combined with the selection below, this will yield the same result and is probably better for the compiler since a while loop with `env` changing types each time is probably not very easy to optimize.